docs/editing-components.md

```diff
@@ -0,0 +1,380 @@
+3. The `paragraph` field uses the basic Wagtail `RichTextBlock`,
+   which results in a field with a multiline WYSIWYG text input.
+4. The `links` field uses another basic Wagtail block, `ListBlock`,
+   which is a special type of block that can hold a variable number of
+   some other block (the `Hyperlink` atom block, in this case).
+5. The `Meta` class defines some properties on the `RelatedContent` block
+   that are used by the Wagtail admin or in rendering the block.
+6. The `icon` property tells Wagtail what icon to use in the editor
+   for the button you use to add a `RelatedContent` block to a StreamField.
+   Icon options can be found in the Wagtail style guide when running locally:
+   http://localhost:8000/admin/styleguide/#icons
+7. The optional `label` property overrides the text of that same button;
+   if `label` is not set, Wagtail will generate one from the name of the block.
+8. The `template` property is a pointer to the HTML template used to render
+   this component. [See below for more on templates.](#the-html-template)
+
+This results in a module that looks like this in the Wagtail editor:
+
+![A screenshot of a Related Content module with heading, paragraph, and links fields](img/related-content-module.png)
+
+Note again that what we think of as **fields** are _also_ blocks,
+and what we think of as **components** or **modules**
+are a special kind of block, `StructBlock`,
+that comprise the sub-blocks that are our fields.
+
+There are two common optional things that are also used in component classes:
+
+1. [Overriding the default `get_context` method](http://docs.wagtail.io/en/v1.13.4/topics/streamfield.html#streamfield-get-context)
+   to pass additional data to the template
+2. [Adding component-specific JavaScript](#adding-some-javascript)
+    via the `Media` class
+
+#### Adding it to a StreamField
+
+Components are made available in the page editing interface
+by adding them to one of a page types's StreamFields.
+These are usually the first things in a page's class definition.
+For example, see this snippet from `blog_page.py`:
+
+```python
+class BlogPage(AbstractFilterPage):
+    content = StreamField([
+        ('full_width_text', organisms.FullWidthText()),
+        ('info_unit_group', organisms.InfoUnitGroup()),
+        ('expandable', organisms.Expandable()),
+        ('well', organisms.Well()),
+        ('email_signup', organisms.EmailSignUp()),
+        ('feedback', v1_blocks.Feedback()),
+        ('image_text_50_50_group', organisms.ImageText5050Group()),
+    ])
+    …
+```
+
+This sets up a StreamField named `content` that allows for
+the insertion of any of those seven listed blocks into it.
+
+To make the `RelatedContent` module (shown above) available to this StreamField,
+we'd add a new entry to this list following the same format:
+`('related_content', molecules.RelatedContent()),`.
+
+Most page types have a couple StreamFields (`header` and `content`) in the
```

Comment:
"a couple of"? Maybe "two" to be more formal?

Reply:
Addressed in 968e783c4997ad98d51c1f8310159a1a68772b07